src/+otp/+arenstorf/+presets/Canonical.m

```diff
@@ -0,0 +1,14 @@
+classdef Canonical < otp.arenstorf.ArenstorfProblem
+    methods
+        function obj = Canonical
+            params.m1 = 0.012277471;
+            params.m2 = 1-0.012277471;
+
+            y0 = [0.994; 0; 0; -2.00158510637908252240537862224];

```

Comment:
That is a very oddly specific y0. Matlab tells me that -2.001585106379082e+00 is at 64-bit machine precision equivalent. I am assuming that there is a closed form expression from which you derived this number? Maybe just keep that (even if it has logs and stuff like that).

Reply:
It's specific because it's the init condition that makes the solution periodic. I couldn't find the closed-form solution for it. I think we should keep the double prec accuracy.

Reply:
It is way more than double precision. If this is the value that is specified in the paper/book I agree that we keep it. It also future-proofs us to extended precision or maybe even quad precision with that many digits.